CHANGELOG.md

```diff
@@ -7,6 +7,58 @@
 
 # Release notes
 
+## 1.0.0
+
+This release switches from Crosswalk to WebView ([#169](https://github.com/medic/cht-android/issues/169)) which brings great benefits, such as:
+- Newer version of Chrome that has better performance, better security, more features and fewer bugs.
+- Drop Crosswalk library that has been unsupported since 2016.
+- Better support for upcoming versions of CHT Core.
+- Smaller APK size. Before it was 746KB, now it is 381KB.
+
+### Upgrade steps
+
+Install CHT Android v1.0.0 in the phone, then upgrade [WebView](https://play.google.com/store/apps/details?id=com.google.android.webview) to the latest available version in Google Play Store and enable automatic updates of this app by following the [Google Documentation](https://support.google.com/googleplay/answer/113412). This will ensure top performance when using CHT.
+
+### Performance improvements
+
+This is the performance test result that compares CHT Android v0.8.0 with the new CHT Android v1.0.0.
+- **Device:** Sony Xperia Z1
+- **Android:** 5.1.1
+- **CHT Core:** 3.15.0
+
+| Test | v0.8.0 | v1.0.0 | Improvement |
+| ------------ | ------------ | ------------ | ------------ |
+| Initial load (fetching 1882 documents) | 0:03:14  | 0:01:35  | **204%** |
+| Refreshing after initial replication | 0:00:10  | 0:00:03  | **333%** |
+| Loading Contacts tab | 0:00:17  |  0:00:12 | **142%** |
+| Loading Reports tab | 0:00:23  | 0:00:21  | **109%** |
+| Loading a specific report (New Pregnancy) | 0:00:04  |  0:00:02 | **200%** |
+| Track opening a form (New Pregnancy) | 0:00:05  | 0:00:04  | **125%** |
+| Loading Tasks tab | 0:00:02  | 0:00:01  | **200%** |
```

Comment:
Sorry, I should have picked this up yesterday, but saying "100% improvement" implies to me that we've doubled performance. Do you think we should subtract 100 off each of these, eg:

```suggestion
| Test | v0.8.0 | v1.0.0 | Improvement |
| ------------ | ------------ | ------------ | ------------ |
| Initial load (fetching 1882 documents) | 0:03:14  | 0:01:35  | **104%** |
| Refreshing after initial replication | 0:00:10  | 0:00:03  | **233%** |
| Loading Contacts tab | 0:00:17  |  0:00:12 | **42%** |
| Loading Reports tab | 0:00:23  | 0:00:21  | **9%** |
| Loading a specific report (New Pregnancy) | 0:00:04  |  0:00:02 | **100%** |
| Track opening a form (New Pregnancy) | 0:00:05  | 0:00:04  | **25%** |
| Loading Tasks tab | 0:00:02  | 0:00:01  | **100%** |
```

Reply:
That makes sense, yesterday I thought on:  (old - new) / new
But then tried to work out the biggest number possible. 
I've applied: (old - new) / new